AUTHORS.md

```diff
@@ -9,4 +9,5 @@ This is the official list of authors for copyright purposes.
 * Marco Favorito <marco.favorito@gmail.com> [marcofavorito](https://github.com/marcofavorito)
 * Jose Moreira Sanchez <jose.moreira.sanchez@valory.xyz> [jmoreira-valory](https://github.com/jmoreira-valory)
 * M.A.P. Karrenbelt <m.a.p.karrenbelt@gmail.com> [Karrenbelt](https://github.com/Karrenbelt)
-* Viraj Patel <vptl185@gmail.com> [angrybayblade](https://github.com/angrybayblade)
\ No newline at end of file
+* Viraj Patel <vptl185@gmail.com> [angrybayblade](https://github.com/angrybayblade)
```

Comment:
just odd how these lines appear identical but somehow are not

Reply:
It's the trailing line which was missing initially and then this happens